java/ql/lib/ext/org.apache.commons.io.model.yml

```diff
@@ -16,4 +16,6 @@ extensions:
       pack: codeql/java-all
       extensible: sinkModel
     data:
+# suggested label is not supported:      - ["org.apache.commons.io", "FileUtils", True, "copyInputStreamToFile", "(InputStream,File)", "", "Argument[0]", "TODO", "ai-generated"]
+# suggested label is not supported:      - ["org.apache.commons.io", "FileUtils", True, "copyToFile", "(InputStream,File)", "", "Argument[0]", "TODO", "ai-generated"] # Not sure what this should be; the input stream is read from here - could be a file, could be memory, could be something else.
```

Comment:
These seem to be writing bytes to a file, regardless of the origin. So I'd consider them `write-file`(whereas the second argument, `File`, should be `create-file`):

```suggestion
      - ["org.apache.commons.io", "FileUtils", True, "copyInputStreamToFile", "(InputStream,File)", "", "Argument[0]", "write-file", "ai-generated"]
      - ["org.apache.commons.io", "FileUtils", True, "copyInputStreamToFile", "(InputStream,File)", "", "Argument[1]", "create-file", "manual"]
      - ["org.apache.commons.io", "FileUtils", True, "copyToFile", "(InputStream,File)", "", "Argument[0]", "write-file", "ai-generated"]
      - ["org.apache.commons.io", "FileUtils", True, "copyToFile", "(InputStream,File)", "", "Argument[1]", "create-file", "manual"]
```